src/atoms/TextInput/TextInput.stories.js

```diff
@@ -84,7 +86,7 @@ storiesOf('TextInput', module)
           </Space>
           <TextInput
             width={select('Width', sizeOptions, 'large', 'Common')}
-            label="Label"
+            label="Label1"
```

Comment:
![image](https://user-images.githubusercontent.com/5961873/79890868-bd1e0280-841d-11ea-8cd4-a1af9039b22d.png)

Reply:
After entering a value, the label comes back again instead of sticking on top.

Reply:
![image](https://user-images.githubusercontent.com/5961873/79893323-8ba73600-8421-11ea-8258-0abd3950223d.png)

Reply:
The colour of label is not correct. Also, bottom spacing of label doesn't look right.

Reply:
Done

Reply:
![image](https://user-images.githubusercontent.com/5961873/79937638-0bf88600-8478-11ea-99b4-ace93d560935.png)

Reply:
Now, it's remaining active/blue even after clicking outside.

Reply:
Also the spacing between value and the input line border is not correct.

Reply:
>Now, it's remaining active/blue even after clicking outside.
Isnt this required ?

>Also the spacing between value and the input line border is not correct. Line height.
I am unable to get exact spacing from figma.

Reply:
No, look at Figma. Blue is there only in active state.